### PR TITLE
Topic/add phone auth

### PR DESCRIPTION
**Pull request by grrrian**
v1.2.0 Add support for Firebase Phone Auth.

### CODE REVIEW

File: src/android/FCMPlugin.java [hunk 1]
```diff
@@ -1,146 +1,256 @@
...
+
+    Log.d(TAG,"==> FCMPlugin execute: "+ action);
+
+    try{
+      // READY //
+      if (action.equals("ready")) {
+        callbackContext.success();
+      }
+      // GET VERIFICATION ID //
+      else if (action.equals("getVerificationID")) {
```
Comment (grrrian) [inline src/android/FCMPlugin.java:61]: This `else if` is the only section I added. The rest is indentation fixes.

File: src/android/FCMPlugin.java [hunk 2]
```diff
@@ -1,146 +1,256 @@
...
+        cordova.getActivity().runOnUiThread(new Runnable() {
```
Comment (BlairElliottVelo) [inline src/android/FCMPlugin.java]: Why the change from the other persons source, where it was `cordova.getThreadPool().execute(new Runnable() { ...`

**Review by BlairElliottVelo — APPROVED** [review]

File: src/android/FCMPlugin.java (hunk 2 above)
Comment (grrrian) [reply src/android/FCMPlugin.java]: good point, let me try that

Comment (grrrian) [reply src/android/FCMPlugin.java]: works!
